pages/page.rb

```diff
@@ -15,6 +15,10 @@ def current_path
         uri.path
     end
 
+    def go_to url
```

Comment:
Possibly could be re-named something like "driver_navigate." Naming it something simple could be a problem if we end up using some other library that has a go_to method

Reply:
Changed to navigate_to instead of go_to, hopefully that's more specific to navigating on a web page